lib/cog/command/service/memory.ex

```diff
@@ -65,54 +70,56 @@ defmodule Cog.Command.Service.Memory do
     # Cleanup processes that died between restarts and monitor processes that
     # are still alive after a restart. If anything dies between restarting and
     # monitoring, the dead process cleanup will catch it.
-    account_for_existing_processes(monitor_table, memory_table)
-    schedule_dead_process_cleanup(@dead_process_cleanup_interval)
+    account_for_existing_pipelines(monitor_table, memory_table, &{&1, :_})
+    schedule_dead_pipeline_cleanup(@dead_pipeline_cleanup_interval)
 
     state = %__MODULE__{memory_table: memory_table, monitor_table: monitor_table}
     {:ok, state}
   end
 
   def handle_call({:fetch, token, key}, _from, state) do
-    result = ets_lookup(state.memory_table, {token, key})
+    result = ETS.lookup(state.memory_table, {token, key})
```

Comment:
Can we change the module name to `EtsWrapper` or something? I think `ETS` will be too confusing over time.